modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -695,6 +762,102 @@ export default class AvgRTPStatsReporter {
     }
 
     /**
+     * Calculates average number of pixels for the report
+     *
+     * @param {map} peerResolutions a map of peer resolutions
+     * @param {boolean} isLocal if the average is to be calculated for the local
+     * video or <tt>false</tt> if for remote videos.
+     * @param {VideoType} videoType
+     * @return {number|NaN} average number of pixels or <tt>NaN</tt> if there
+     * are no samples.
+     * @private
+     */
+    _calculateAvgVideoPixels(peerResolutions, isLocal, videoType) {
```

Comment:
This method is an adaption of what's already there for the FPS. I'm not so enthusiast to change it, since what's there appears to be working well, unless of course you see a problem with the current implementation.